docs/pipelines.md

```diff
@@ -1721,9 +1720,34 @@ Pipelines do not support the following items with custom tasks:
 
 For a better understanding of `Pipelines`, study [our code examples](https://github.com/tektoncd/pipeline/tree/main/examples).
 
+## Known Custom Tasks
```

Comment:
sort of unfortunate that these all implement v1alpha1.Run... maybe we could have one section for "old" custom tasks implementing v1alpha1.Run and one (empty) section for those implementing customrun?